src/app/backend/handler/apihandler.go

```diff
@@ -996,6 +1007,55 @@ func (apiHandler *APIHandler) handleGetPods(
 	response.WriteHeaderAndEntity(http.StatusCreated, result)
 }
 
+var upgrader = websocket.Upgrader{}
+
+func (apiHandler *APIHandler) handleExecIntoPod(request *restful.Request, response *restful.Response) {
+	conn, err := upgrader.Upgrade(response.ResponseWriter, request.Request, nil)
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
+	// For example about how kubexec works see: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/exec.go
+	// req := apiHandler.client.Post().
+	// 	Resource("pods").
+	// 	Name(pod.Name).
+	// 	Namespace("default").
+	// 	SubResource("exec").
+	// 	Param("container", containerName)
+
+	// config := &restclient.Config{}
+	// exec, err := remotecommand.NewExecutor(config, "POST", req.URL())
```

Comment:
I need to create a new executor which is like an API client. I need to pass it a restclient.Config. Not sure the best way to get the config from the existing API client to set up this Config object.